packages/adapter-yb-pg/README.md

```diff
@@ -35,29 +37,31 @@ npx prisma generate
 
 ### 2. Install the dependencies
 
-Next, install the `pg` package and Prisma ORM's driver adapter:
+Next, install the `@yugabytedb/pg` package and Prisma ORM's driver adapter:
 
 ```
-npm install pg
-npm install @prisma/adapter-pg
+npm install @yugabytedb/pg
+npm install path/to/yugabytedb-adapter-yb-pg-6.8.2-yb-1.tgz
 ```
+> **Note:**: Need to install @yugabytedb/pg version 8.7.3-yb-10 or later.
 
 ### 3. Instantiate Prisma Client using the driver adapter
 
 Finally, when you instantiate Prisma Client, you need to pass an instance of Prisma ORM's driver adapter to the `PrismaClient` constructor:
 
 ```ts
-import { PrismaPg } from '@prisma/adapter-pg'
+import { PrismaPg } from '@yugabytedb/adapter-yb-pg'
```

Comment:
Will it be even better to name this `@yugabytedb/prisma-adapter`? From the name itself, it tells it's an adapter for prisma from yugabytedb. I think other external adapters have named their adapters similarly.

Reply:
Yes.